dispatch/static/manager/src/styles/components/header.scss

```diff
@@ -25,3 +25,30 @@
   box-shadow: none;
   border: none;
 }
+
+.event-badge{
```

Comment:
Can you change this classname to `.c-banner__badge`, just to make it a bit more reusable?